proficio_sandbox/configurations/wam3.conf.right

```diff
@@ -0,0 +1,113 @@
+# Configuration file for a 3-DOF robot - Proficio right-handed setup
+wam3:
+{
+	low_level:
+	{
+		# Home position and zero-angle calibration data
+		@include "calibration_data/wam3/zerocal.conf"
+		
+		j2mp = ((    -8.51,  15.19643,     0 ),
+		        (     8.51,  15.19643,     0 ),
+	   	        (        0,         0,   9.55 ));
+		joint_encoder_counts = ( 655360, 655360, 327680);
+	};
+
+	# Calibrated gravity compensation data
+	@include "calibration_data/wam3/gravitycal.conf"
+	
+	kinematics:
+	{  
+		#Setup for WTB centered on person		
+		world_to_base = ((1, 0,  0,  0.00),
+                     (0, 0,  1, -0.540),
+                     (0,-1,  0,  0.00),
+		                 (0, 0,  0,  1.00)
+    );
+			            
+		moving:
+		(
+			# Note: alpha_pi = alpha / pi 
+			{ alpha_pi =  0.5; a =    0.0; d =      0; }, # Base Pitch
+			{ alpha_pi =  0.0; a =  0.650; d =    0.0; }, # Twist
+			{ alpha_pi = -0.5; a =  0.440; d =  0.234; }  # Elbow
+		);
+		toolplate = { alpha_pi = 0; theta_pi = 0; a = 0; d = 0.0; };
+	};
+	
+	dynamics:
+	{
+		/* From inertial specifications */
+		moving:
+		(
+			
+			{
+				# Link 1
+				mass = 3.8749;
+				com = ( -2.37e-3, 31.06e-3, 15.42e-3 );
+				I = (( 21409.58e-6,   271.72e-6,    24.61e-6 ),
+					 (   271.72e-6, 13778.75e-6, -1819.20e-6 ),
+					 (    24.61e-6, -1819.20e-6, 15589.06e-6 ));
+			},
+			{
+				# Link 2
+				mass = 1.8023;
+				com = ( -38.26e-3, 207.51e-3, 0.03e-3 );
+				I = (( 59110.77e-6, -2496.12e-6,     7.38e-6 ),
+					 ( -2496.12e-6,  3245.50e-6,   -17.67e-6 ),
+					 (     7.38e-6,   -17.67e-6, 59270.43e-6 ));
+			},
+			{
+				# Link 3
+				mass = 1.0651;
+				com = ( 10.95e-3, -0.03e-3, 140.54e-3 );
+				I = (( 18485.77e-6,     2.19e-6, -1608.68e-6 ),
+					 (     2.19e-6, 18916.58e-6,     5.15e-6 ),
+					 ( -1608.68e-6,     5.15e-6,  1975.17e-6 ));
+			}
+
+			# Tool?
+		);
+	};
+
+	joint_velocity_filter:
+	{
+		type = "low_pass";
+		omega_p = (180, 180, 180);
+	};
+
+	joint_position_control:
+	{
+		kp = (1200, 2000, 900);
```

Comment:
Same as other conf file: We should tune these to reduce oscillations. kp=(1200,2000,500) worked better for me.